### DISCUSSION TITLE
Plugin Install fails With several errors/timeouts

### DISCUSSION THREAD

**Discussion by thetaz001** [richierockskool/homebridge-inkbird-wifi-gateway#26] [Q&A]
Looking to install this plugin..
and when i attempt to install it, i get this
![image](https://github.com/richierockskool/homebridge-inkbird-wifi-gateway/assets/84531926/c9676b5c-db56-4be7-9c1c-9fead82aba34)
the config page comes up, i do enter my information for my inkbird device, but it never comes up, and IN the status page when the plugins start i get this
[7/31/2023, 4:26:02 PM] No plugin was found for the platform "HomebridgeInkbirdWifiGateway" in your config.json. Please make sure the corresponding plugin is installed correctly.
any help would be much appreciated.
fred

**Comment by richierockskool** [comment]
try again, i was updating

**Comment by thetaz001** [comment]
unfortunately, no still same issue
![image](https://github.com/richierockskool/homebridge-inkbird-wifi-gateway/assets/84531926/87a48265-679d-4bf0-842d-88b617a8cf73)

**Reply by richierockskool** [reply]
It is installing ....

CMD: npm install -g homebridge-inkbird-wifi-gateway@1.2.1

[##################] \ reify:web-streams-polyfill: timing reifyNode:node_modules/homeb
> @abandonware/noble@1.9.2-23 install
> node-gyp-build

  CC(target) Release/obj.target/nothing/../../node-addon-api/nothing.oidge-inkbird-wif
  LIBTOOL-STATIC Release/nothing.atiming reifyNode:node_modules/homebridge-inkbird-wif
warning: /Library/Developer/CommandLineTools/usr/bin/libtool: archive library: Release/nothing.a the table of contents is empty (no object file members in the library define global symbols)
  CXX(target) Release/obj.target/binding/lib/mac/src/noble_mac.o
[##################] - reify:usb: timing reifyNode:

**Reply by thetaz001** [reply]
It does look like it appears to, but then i get those errors..
404 not found, and the plug in does not function, when i restart homebridge ?? :-/

**Reply by thetaz001** [reply]
And when i restart homebridge i get this
[8/1/2023, 4:48:40 PM] No plugin was found for the platform "HomebridgeInkbirdWifiGateway" in your config.json. Please make sure the corresponding plugin is installed correctly.

**Comment by richierockskool** [comment]
That is what we are working on, hopefully success soon.

**Reply by thetaz001** [reply]
AHHH oh ok, gothcya, sounds good.. I am happy to test as soon as you are ready for me too..
thanks so much for working on and fixing this :)

Fred

**Comment by richierockskool** [comment]
You can copy and paste this config Json in the mean time as least the plugin will be found,
enter your email and password from the new inkbird app.

{
            "name": "InkbirdWifiGateway",
            "email": “xxxxxxxxx",
            "password": “xxxxxxxx",
            "host": "99.228.219.159",
            "mac_address": "e8:db:84:b6:9c:e6",
            "showBridge": true,
            "showTempSensor": true,
            "useTemperatureDisplay": true,
            "update_interval": 5,
            "loglevel": 4,
            "platform": "homebridge-inkbird-wifi-gateway"
        }

**Reply by thetaz001** [reply]
Was wondering on the status of this?
i had time today to finally try and add this config to my json config file, and when i try to save it says config syntax error
config contains invalid JSON
